handsontable/src/editors/dateEditor/dateEditor.js

```diff
@@ -138,10 +151,16 @@ export class DateEditor extends TextEditor {
       this.$datePicker.destroy();
     }
 
-    this.instance._registerTimeout(() => {
-      this.instance._refreshBorders();
+    this.hot._registerTimeout(() => {
+      this.hot._refreshBorders();
     });
 
+    if (this.TD && this.hot.getSettings().ariaTags) {
```

Comment:
```suggestion
    if (this.hot.getSettings().ariaTags) {
```
Isn't the TD element always available?

Reply:
Thanks for explanation 👍